app/mesh/1.5.x/features/vault.md

```diff
@@ -177,7 +185,10 @@ Vault, you must provide credentials in the configuration of the `mesh` object of
 You can authenticate with the `token` or with client certificates by providing `clientKey` and `clientCert`.
 
 You can provide these values inline for testing purposes only, as a path to a file on the
-same host as `kuma-cp`, or contained in a `secret`. See [the Kuma Secrets documentation](https://kuma.io/docs/latest/security/secrets/).
+same host as `kuma-cp`, or contained in a `secret`. When using a `secret`, it should be a mesh-scoped 
+secret (see [the Kuma Secrets documentation](https://kuma.io/docs/latest/security/secrets/) for details 
+on mesh-scoped secrets versus global secrets). On Kubernetes, this mesh-scoped secret should be stored 
+in the system namespace ("kong-mesh-system" by default).
```

Comment:
I suppose we might as well call out `type: system.kuma.io/secret` here

Reply:
I can add that to the end of the paragraph where I mention that it needs to be found in the "kong-mesh-system" namespace.